tests/src/PatternAssertions/ListWithIllustrationAssert.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\PatternAssertions;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Assertions for the list with illustration pattern.
+ */
+class ListWithIllustrationAssert extends BasePatternAssert {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getAssertions($variant): array {
+    return [
+      'column' => [
+        [$this, 'assertColumns'],
+        $variant,
+      ],
+      'zebra' => [
+        [$this, 'assertElementExists'],
```

Comment:
If `zebra` isn't enabled in pattern, then `.ecl-list-illustration--zebra` is absent. Not sure that simple `assertElementExists` can process it.